CMakeLists.txt

```
@@ -24,18 +24,18 @@ option(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY "Build with file/folder reorg wit
 
 rocm_setup_version(VERSION 1.12.0 NO_GIT_TAG_VERSION)
 
-install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/include/half.hpp DESTINATION include/half)
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/include/half.hpp DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/half)
 
 if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY)
   include(ROCMHeaderWrapper)
   rocm_wrap_header_file(
     half.hpp
     GUARDS WRAPPER
-    WRAPPER_LOCATIONS include
+    WRAPPER_LOCATIONS ${CMAKE_INSTALL_INCLUDEDIR}
   )
 
   #Install Wrapper for backward compatibility
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/include/half.hpp DESTINATION include)
+  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/include/half.hpp DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
```

As `OUTPUT_LOCATIONS` is unspecified in the call to `rocm_wrap_header_file`, it uses the value of `WRAPPER_LOCATIONS`, so the wrapper file will be output to `${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_INCLUDEDIR}/half.hpp`.
```suggestion
  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_INCLUDEDIR}/half.hpp DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
```
The other option is to add `OUTPUT_LOCATIONS include` to the `rocm_wrap_header_file` call on line 31.